### ISSUE TITLE
file not found error.

### ISSUE THREAD

**Issue by rajucse3040** [Wizcorp/phonegap-facebook-plugin#956]
I have been using the Xcode 6.1.1 and i'm using the Facebook. So i'm getting the below error 

'FacebookSDK/FacebookSDK.h' file not found

FYI :  I have tried the $(Project_Dir)/ProjectName

**Comment by edvella** [comment]
There's a warning on the project readme that explains this problem:

<< --- Cordova Registry Warning [iOS]

*Installing this plugin directly from Cordova Registry results in Xcode using a broken FacebookSDK.framework, this is because the current publish procedure to NPM breaks symlinks CB-6092. Please install the plugin through a locally cloned copy or re-add the FacebookSDK.framework to Xcode after installation.*

------------------------------------------ >>

Basically you should copy the plugin to a local folder and install that one using:
cordova plugin add [path-to-local-folder]

**Comment by bairathirahul** [comment]
For those who are still facing this issue, upgrade the FacebookSDK.framework to version 3.23

**Comment by fedepasi** [comment]
how could i update the facebookSDK.framework ?

**Comment by bairathirahul** [comment]
I don't remember actually, it has been 2 years. However, try replacing the file phonegap-facebook-plugin/platforms/ios/FacebookSDK.framework with the latest version downloaded from here:

https://developers.facebook.com/docs/ios/
